tox.ini

```diff
@@ -22,12 +23,14 @@ deps =
 	djangomaster: https://github.com/django/django/archive/master.tar.gz
 	djangorestframework
 	oauthlib>=3.0.1
+	jwcrypto
 	coverage
 	pytest
 	pytest-cov
 	pytest-django
 	pytest-xdist
 	py27: mock
+	jwcrypto
```

Comment:
I seem like a duplicate change with the line previous.

Reply:
Fix in https://github.com/jazzband/django-oauth-toolkit/pull/545/commits/96da3635c5ab5f917cb553f13455c71cc0f7a867